amivapi/settings.py

```diff
@@ -116,3 +116,37 @@
     'mtec',
     'gess'
 ]
+
+# Config for swagger (/docs)
+# TODO: should be linked to the setup.py information in the package
+SWAGGER_INFO = {
+    'title': 'AMIVAPI',
+    'version': '0.1-dev',
```

Comment:
We should get this from a constant somewhere to make it automatically update over releases.